spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -596,4 +596,33 @@ As specified in <<builtin_contexts>>, contexts associated with built-in normal s
 * Otherwise, if the observer method is any other kind of transactional observer method, it is called in an unspecified transaction context, but with the same lifecycle contexts as the transaction that just completed.
 * Otherwise, the observer method is called in the same transaction context and lifecycle contexts as the invocation of `Event.fire()` or `BeanManager.getEvent()`.
 
+[[observable_container_lifecycle_events]]
+
+=== Observable container lifecycle events
```

Comment:
Since "Container lifecycle events" belong to Portable Extensions, should we call these differently? Maybe "Application lifecycle events"? The Packaging and Deployment chapter has sections "Application initialization lifecycle" and "Application shutdown lifecycle", which is a nice match. (These sections don't have to refer to these new event types, as you described.)